README.md

```diff
@@ -8,9 +8,11 @@ Documentation is in progress [over here](https://aced-differentiate.github.io/At
 
 ## Getting Started
 
-1. Clone this package to wherever you want to play.
+1. To install the latest tagged version, in your Julia REPL, do `]install AtomicGraphNets`. However, you can also play with the latest version on the `main` branch by skipping to step 2 and then doing `]install /path/to/repo` where you replace the dummy path with the location of your clone.
 
```

Comment:
`] install`? shouldn't this be `] add` or is there something I'm missing?

Reply:
hahaha yup oops